suite-common/wallet-utils/src/sendFormUtils.ts

```diff
@@ -204,10 +203,7 @@ export const getFeeUnits = (networkType: NetworkType) => {
     return 'sat/B';
 };
 
-export const getFee = (
-    networkType: NetworkType,
-    tx: PrecomposedTransactionFinal | TxFinalCardano,
-) => {
+export const getFee = (networkType: NetworkType, tx: GeneralPrecomposedTransactionFinal) => {
```

Comment:
Oh, I forgot to answer to this no. I would rather not touch this. If I change the parameteres of this function, I think I should also change the parameters of all the other functions of this utils file. ALso this function is impoted to many places, so it would create a huge diff. 

I got your point and agree with you, but I would rather avoid this.